core/system_metric/sys_metric_stat.go

```diff
@@ -32,6 +38,9 @@ const (
 	NotRetrievedLoadValue     float64 = -1.0
 	NotRetrievedCpuUsageValue float64 = -1.0
 	NotRetrievedMemoryValue   int64   = -1
+	CGroupPath                        = "/proc/1/cgroup"
+	DockerPath                        = "/docker"
```

Comment:
What if the container runtime is not Docker? Kubernetes is deprecating Docker as a container runtime after v1.20. Refer https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/

Reply:
![image](https://user-images.githubusercontent.com/35391181/125226388-93dbe180-e303-11eb-80d2-6f2d44bafe04.png)
the next line is the path in Kubernetes.